src/Microsoft.Azure.EventHubs/Primitives/ClientInfo.cs

```diff
@@ -23,7 +23,7 @@ static ClientInfo()
                 product = GetAssemblyAttributeValue<AssemblyProductAttribute>(assembly, p => p.Product);
                 version = GetAssemblyAttributeValue<AssemblyVersionAttribute>(assembly, v => v.Version);
                 framework = GetAssemblyAttributeValue<TargetFrameworkAttribute>(assembly, f => f.FrameworkName);
-#if NETSTANDARD
+#if NETSTANDARD1_3
```

Comment:
Sean, generalized preprocessor directive is only for this case. For other possible cases I agree with you. But here we are looking a code snippet that applies to all NetStandard versions.